lib/logmsg/logmsg.c

```diff
@@ -1089,6 +1089,7 @@ log_msg_init(LogMessage *self, GSockAddr *saddr)
   self->timestamps[LM_TS_RECVD].tv_usec = tv.tv_usec;
   self->timestamps[LM_TS_RECVD].zone_offset = get_local_timezone_ofs(self->timestamps[LM_TS_RECVD].tv_sec);
   self->timestamps[LM_TS_STAMP] = self->timestamps[LM_TS_RECVD];
+  self->timestamps[LM_TS_PROCESSED] = self->timestamps[LM_TS_RECVD];
```

Comment:
Let's initialize this to something else, like -1 so not to suggest it is an alias. Other than that, this patch looks good to me.